README.md

```diff
@@ -34,6 +39,31 @@ It is possible to send args to the script under test by adding -- after the load
 ./bin/load-runner.js -s ./scripts/dummy.js -c 5 -n 100 -r 10 -o -- -testarg1 testval1
 ```
 
+# Environment variables
+
+There are some environment variables generated for the test script that can be useful. You can use Node.js `process.env` object to access them.
+
+* `LR_FLOW_NUMBER` - number generated each run, value depends on percentages specified by `-f` parameter, see [Flow numbers](#flow-numbers)  
```

Comment:
Yes, it's useful to be able to run them standalone (since that's what load-runner does anyway, it just spawns a new node process with them). It's still possible with this change, you just need to keep in mind that the env vars need to be set if not running with load-runner.